deploy/deploy-legion-enclave.Jenkinsfile

```diff
@@ -11,16 +11,20 @@ node {
         
         stage('Deploy Legion Enclave') {
             dir('deploy/ansible'){
-                withCredentials([file(credentialsId: params.Profile, variable: 'CREDENTIAL_SECRETS')]) {
+                withCredentials([
+                    file(credentialsId: "vault-${params.Profile}", variable: 'vault'),
+                    string(credentialsId: 'vault_bucket', variable: 'vault_bucket')]) {
                     withAWS(credentials: 'kops') {
                         wrap([$class: 'AnsiColorBuildWrapper', colorMapName: "xterm"]) {
                             ansiblePlaybook(
                                 playbook: 'deploy-legion-enclave.yml',
-                                extras: ' --extra-vars "profile=${Profile} base_version=${BaseVersion}  local_version=${LocalVersion} enclave_name=${EnclaveName}"',
+                                extras: '--vault-password-file=${vault} \
+                                        --extra-vars "profile=${Profile} \
+                                        bucket=${vault_bucket} base_version=${BaseVersion}  \
```

Comment:
Why do we want to parameterize secrets bucket? Guess this should be linked to profile to avoid mistakes or misuse.

Reply:
bucket name is unique name in scope of AWS region and could be equated to sensitive information

Reply:
can't really imagine how s3 bucket could be compromised... By the way we have the bucket name for each cluster in profile in git right now, so reusing the same location doesn't bring any extra risks (if there are any)